packages/strapi-admin/admin/src/containers/App/tests/actions.test.js

```diff
@@ -0,0 +1,141 @@
+import {
+  FREEZE_APP,
+  GET_APP_PLUGINS_SUCCEEDED,
+  LOAD_PLUGIN,
+  PLUGIN_DELETED,
+  PLUGIN_LOADED,
+  UNFREEZE_APP,
+  UNSET_HAS_USERS_PLUGIN,
+  UPDATE_PLUGIN,
+} from '../constants';
+import {
+  freezeApp,
+  getAppPluginsSucceeded,
+  loadPlugin,
+  pluginDeleted,
+  pluginLoaded,
+  unfreezeApp,
+  unsetHasUserPlugin,
+  updatePlugin,
+} from '../actions';
+
+describe('<App /> actions', () => {
+  describe('freezeApp', () => {
+    it('should return the correct type and the passed data', () => {
```

Comment:
I'm not sure if the naming convention is right... it's too generic. Why are you not naming this test `should freeze the app when the freezeApp action is called`?

See https://stackoverflow.com/questions/155436/unit-test-naming-best-practices

Reply:
Because it doesn't freeze the app the reducer props does which is tested in a separated file.

Reply:
It wrapped in a describe so I believe that the naming is correct.